src/evaluator.ts

```diff
@@ -62,7 +54,10 @@ export class Evaluator {
 
   async batchEvaluation(evaluationContext: EvaluationContext) {
     const results = await this.flagrEvaluation(evaluationContext);
-    const cachedEvaluate = <T>(flag: string, callbacks: FlagCallbacks<T>) => {
+    const cachedEvaluate = <T>(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏Can probably just drop `cached` from this and match below